lib/syskit/telemetry/ui/runtime_state.rb

```diff
@@ -645,20 +650,37 @@ def select_job(job_status)
                     job_expanded_status.add_tasks_info(all_tasks, all_job_info)
                 end
 
-                def restore_from_settings(settings)
+                def settings
+                    @settings ||= Qt::Settings.new("syskit", "telemetry-ui")
+                end
+
+                def restore_from_settings(settings = self.settings)
                     %w{ui_hide_loggers ui_show_expanded_job}.each do |checkbox_name|
                         default = Qt::Variant.new(send(checkbox_name).checked)
                         send(checkbox_name).checked = settings.value(checkbox_name, default).to_bool
                     end
                 end
 
-                def save_to_settings(settings)
+                def save_to_settings(settings = self.settings)
                     %w(ui_hide_loggers ui_show_expanded_job).each do |checkbox_name|
                         settings.set_value checkbox_name, Qt::Variant.new(send(checkbox_name).checked)
                     end
                 end
 
                 signals "fileOpenClicked(const QUrl&)"
+
+                def self.exec(host, port: Roby::Interface::DEFAULT_PORT)
+                    Orocos.initialize
+                    interface = Roby::Interface::V1::Async::Interface.new(host, port: port)
```

Comment:
V1?

Reply:
"copy of the current runtime view" -> still V1. https://github.com/rock-core/tools-syskit/pull/401 converts to V2